OpenRA.Mods.Common/Activities/DonateCash.cs

```diff
@@ -42,6 +42,12 @@ protected override void OnInside(Actor self)
 
 			if (self.Owner.IsAlliedWith(self.World.RenderPlayer))
 				self.World.AddFrameEndTask(w => w.Add(new FloatingText(target.CenterPosition, target.Owner.Color.RGB, FloatingText.FormatCashTick(payload), 30)));
+
+			foreach (var host in target.TraitsImplementing<INotifyCashTransfer>())
```

Comment:
nit: the variable is a trait, not an actor, so `host` isn't an appropriate name.
Our convention for these queries is to use an abbreviation of the interface name, so `var ntc` here.